Mathlib/MeasureTheory/Integral/IntervalIntegral.lean

```diff
@@ -90,6 +92,23 @@ theorem IntervalIntegrable.congr {g : ℝ → E} (hf : IntervalIntegrable f μ a
     IntervalIntegrable g μ a b := by
   rwa [intervalIntegrable_iff, ← integrableOn_congr_fun_ae h, ← intervalIntegrable_iff]
 
+/-- Interval integrability is invariant when functions change along discrete sets. -/
+theorem intervalIntegrable_congr_codiscreteWithin {g : ℝ → E}
```

Comment:
instead of stating that just for the volume, can you state it for an arbitrary measure without atoms?

Also, could you give first an implication (named `IntervalIntegrable.congr_codiscreteWithin`) and then the iff version as you do? Note that the implication has a one-line proof, as 
```
theorem IntervalIntegrable.congr_codiscreteWithin {g : ℝ → E}
    (h : f =ᶠ[codiscreteWithin (Ι a b)] g) 
    (hf : IntervalIntegrable f volume a b) : IntervalIntegrable g volume a b :=
  hf.congr (ae_of_restrVol_le_codiscreteWithin measurableSet_Ioc h)
```

Reply:
Done.